nix/runtime.nix

```diff
@@ -20,13 +20,26 @@ rec {
       tag = "1.35.4";
     };
     ogmios = { port = 1337; };
+    postgres = {
```

Comment:
please add docker image tag option

Reply:
I will use a Nix generated Docker image to improve reproducibility.